traffic_ops/client/stats_summary.go

```diff
@@ -72,6 +72,6 @@ func (to *Session) CreateSummaryStats(statsSummary tc.StatsSummary) (tc.Alerts,
 	if err != nil {
 		return tc.Alerts{}, reqInf, err
 	}
-	reqInf, err = post(to, fmt.Sprintf("%s/%s", basePathSummaryStats, "create"), reqBody, &alerts)
+	reqInf, err = post(to, basePathSummaryStats, reqBody, &alerts)
```

Comment:
If you're going to change this client method then it needs to exactly match the behavior of the old endpoint. This one doesn't, because `summaryTime` and `statDate` aren't optional anymore.

Reply:
statDate is optional and summaryTime has never been optional in my testing I get a 500 when testing it on the old stats_summary/create if I dont pass it in

Reply:
nvm talked to @ocket8888 offline and he pointed at the differences that I need to match

Reply:
Okay looking at this a bit I dont think this needs to match the behavior of the old endpoint as we decided on the mailing list we do not need deprecation periods for TO client methods. 

The problem is to match the old behavior you are seeing I would have to change the underlying struct (or make a new one) to take in strings instead of time.Time as you pointed out strings like "today", "yesterday" work. 

I think leaving this as is is fine imo. I guess I could make another client method with a different struct input and post to the old route. Thoughts?

Reply:
> _"...we decided on the mailing list we do not need deprecation periods for TO client methods."_

That's good enough for me.